service_mesh/v2x/ossm-gateway-migration.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-gateway-migration"]
+= Gateway Migration
+include::_attributes/common-attributes.adoc[]
+:context: gateway-migration
+
+toc::[]
+
+As a network administrator, the preferred method for deploying ingress and egress gateways is to do so with a `Deployment` resource using gateway injection.
+
+include::modules/ossm-about-gateway-migration.adoc[leveloffset=+1]
+
+include::modules/ossm-migrating-from-smcp-defined-gateways-to-gateway-injection.adoc[leveloffset=+1]
+
+.Additional resources
+
+xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-automatic-gateway-injection_traffic-management[Enabling gateway injection]
+
+xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-deploying-automatic-gateway-injection_traffic-management[Deploying automatic gateway injection]
```

Comment:
Also, these xrefs need to be list items:
```suggestion
* xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-automatic-gateway-injection_traffic-management[Enabling gateway injection]

* xref:../../service_mesh/v2x/ossm-traffic-manage.adoc#ossm-deploying-automatic-gateway-injection_traffic-management[Deploying automatic gateway injection]
```

Reply:
done